### ISSUE TITLE
Windows: self-extract progress bar never closes until namebench exits

### ISSUE THREAD

**Issue by GoogleCodeExporter** [jeevandongre/namebench#30]
```
1. Open namebench-1.0.exe in Windows
2. Progress bar window opens
3. namebench.exe opens once progress bar reaches 100%
4. Progress bar stays in background until namebench.exe is closed.

This is an artifact of the Zip 2 Secure EXE packaging we use. This bug will 
either need to be fixed on their end or a suitable replacement is found.

```

Original issue reported on code.google.com by `tstromb...@google.com` on 5 Dec 2009 at 7:14

**Comment by GoogleCodeExporter** [comment]
```
[deleted comment]
```

**Comment by GoogleCodeExporter** [comment]
```
[deleted comment]
```

**Comment by GoogleCodeExporter** [comment]
```
[deleted comment]
```

**Comment by GoogleCodeExporter** [comment]
```
Issue 55 has been merged into this issue.
```

Original comment by `tstromb...@google.com` on 8 Dec 2009 at 3:15

**Comment by GoogleCodeExporter** [comment]
```
Can anyone recommend a self-extracting EXE builder for Windows that can launch 
an 
.exe at completion that does not have this bug? 

```

Original comment by `tstromb...@google.com` on 8 Dec 2009 at 3:16
- Changed title: **Windows: self-extract progress bar never closes until namebench exits**

**Comment by GoogleCodeExporter** [comment]
```
Fixed in version 1.0.3 buy using FreeExtractor.

```

Original comment by `thomas.r...@gmail.com` on 8 Dec 2009 at 5:52
- Changed state: **Fixed**
